deploy/crds/org_v1_che_crd.yaml

```diff
@@ -220,7 +220,7 @@ spec:
                 used by the Che installation.
               properties:
                 enable:
-                  description: Enables `metrics` Che server endpoint.
+                  description: Enables `metrics` Che server endpoint. Default to `false`.
```

Comment:
Does this change belong in this PR?

Reply:
no it doesn't. It's generated code. I've forgot to generate latest changes in previous PR. As it is just description, I've leave it here. I can do it in separate PR if you think it would be better.